Add troubleshooting section to `concept-private-endpoint.md` (#125995)
Azure/azure-content: articles/azure-app-configuration/concept-private-endpoint.md

Enabling private endpoints requires a [Standard or Premium tier](https://azure.microsoft.com/pricing/details/app-configuration/) App Configuration store. To learn about private link pricing details, see [Azure Private Link pricing](https://azure.microsoft.com/pricing/details/private-link).

## Troubleshooting private endpoint errors

> ### Troubleshoot RP registration errors

Reviewer: ```suggestion
### Troubleshoot resource provider registration errors
```

Author: Updated, thanks!

The following error indicates that the private endpoint being connected to an App Configuration store is in a subscription that has not registered the Azure App Configuration resource provider:

> > The private endpoint's subscription '00000000-0000-0000-0000-000000000000' is not registered to use resource provider 'Microsoft.AppConfiguration'. See https://aka.ms/appconfig/PrivateEndpointTroubleshooting for instructions on how to register a subscription to a resource provider.

Reviewer: Suggestions:
* Replace this fake GUID with the following one from our list of approved GUIDs:
* Use a site-relative link instead of an aka.ms link. For example (./concept-private-endpoint.md#troubleshooting-private-endpoint-errors)
* This link sends customers to the section they're already reading. If this is where you want to send customers, I suggest removing the sentence, otherwise update the link to another section or article.

```suggestion
> The private endpoint's subscription 'aaaa0a0a-bb1b-cc2c-dd3d-eeeeee4e4e4e' is not registered to use resource provider 'Microsoft.AppConfiguration'.
```

Author: Thanks for the suggestions, all have been addressed.

> This error is typically seen when the private endpoint's subscription is different to the App Configuration store's subscription. To resolve:

Reviewer: ```suggestion
This error is typically seen when the private endpoint's subscription is different from the App Configuration store's subscription. To resolve:
```

1. Register the `Microsoft.AppConfiguration` resource provider in the private endpoint's subscription.

> 2. Reconnect the private endpoint to the App Configuration store.

Reviewer: All steps of our numbered lists start with "1." on Microsoft Learn and are automatically rendered as 1. 2. 3.

```suggestion
1. Reconnect the private endpoint to the App Configuration store.
```

> 1. Register the private endpoint's subscription to the `Microsoft.AppConfiguration` resource provider.

Reviewer 2: ```suggestion
1. Register the `Microsoft.AppConfiguration` resource provider in the private endpoint's subscription.
```

Author: Updated, resolving

Learn to configure your DNS server with private endpoints:

- [Name resolution for resources in Azure virtual networks](../virtual-network/virtual-networks-name-resolution-for-vms-and-role-instances.md#name-resolution-that-uses-your-own-dns-server)

> - [DNS configuration for Private Endpoints](../private-link/private-endpoint-overview.md#dns-configuration)

Reviewer: ```suggestion
- [DNS configuration for private endpoints](../private-link/private-endpoint-overview.md#dns-configuration)
```
